src/de.hpi.swa.graal.squeak.test/src/de/hpi/swa/graal/squeak/test/tests.properties

```diff
@@ -699,7 +699,7 @@ ColorTest>>testMultiplyByNumber=passing
 ColorTest>>testNew=passing
 ColorTest>>testPrintHtmlString=passing
 ColorTest>>testUnCategorizedMethods=passing
-CompiledMethodComparisonTest>>testHash=passing
+CompiledMethodComparisonTest>>testHash=not_terminating
```

Comment:
I will add the info as a comment to the properties file, so the next time we look at this, we know why it's marked as `not_terminating`.